ui/nuxeo-document-permissions/nuxeo-popup-permission.js

```diff
@@ -407,17 +407,18 @@ import '../widgets/nuxeo-user-suggestion.js';
       }
     }
 
-    _getResetParams() {
-      return {
+    _getParams() {
+      const resetParams = {
         users: [],
         username: null,
         email: null,
-        permission: 'Read',
+        permission: this.i18n(this.userVisiblePermissions[0]),
```

Comment:
The `userVisiblePermissions` contains the correct key format? Won't we need to use a format method? (e.g. `formatPermission`)

Reply:
Actually we can set it to null, since there is already a function that is formatting the permissions.